source/Octopus.Tentacle/Commands/RegisterMachineCommandBase.cs

```diff
@@ -218,7 +218,7 @@ Uri GetActiveTentacleAddress()
                     serverCommsPort ??= serverCommsAddressUri.Port;
                 }
 
-                return new Uri($"https://{serverCommsAddressUri.Host}:{serverCommsPort}");
+                return new Uri($"https://{serverCommsAddressUri?.Host}:{serverCommsPort}");
```

Comment:
I'm assuming this is possibly null as `api.ServerUri` might be null? If so, would it not be better to throw earlier and make sure it's not null at this point? having a null host here doesn't make sense

Reply:
Good point 👍 Fixed.